proto/interchain_security/ccv/v1/wire.proto

```diff
@@ -88,6 +77,12 @@ enum ConsumerPacketDataType {
       [ (gogoproto.enumvalue_customname) = "VscMaturedPacket" ];
 }
 
+// Note this type is used during IBC handshake methods for both the consumer and provider
+message HandshakeMetadata {
+  string provider_fee_pool_addr = 1;
+  string version = 2;
+}
```

Comment:
Is my understanding correct that the reward distribution is performed through a `MsgTransfer` [message](https://github.com/cosmos/interchain-security/blob/cca008d856e3ffc60ec1a486871d0faa702abe26/x/ccv/consumer/keeper/distribution.go#L126) and hence why we do not see something reward-related in this file? If yes, it might make sense to add a comment here to point to this.

Reply:
Yea consumer rewards are sent through a distribution channel, whereas wire.proto is for schemas sent over a ccv channel, [this comment](https://github.com/cosmos/interchain-security/commit/124803a95e5246bba5134baaf2378411a5b0053e) can hopefully help